bin/losant.js

```diff
@@ -7,6 +7,7 @@ const pgm = program
   .version(pkg.version)
   .description('Losant CLI is a command line tool to help manage your Losant Application and its resources.')
   .command('login', 'Log in and create your user configuration to use the other commands.')
+  .command('set-token', 'Retrieve a User API Token from your Losant Application to set your user configuration.')
```

Comment:
Not sure about this phrasing; you don't get a token from a Losant "application."
```suggestion
  .command('set-token', 'Create a User API Token in your Losant account, then set it here to configure the command line tool.')
```

Reply:
don't you want to have the same text here as here: https://github.com/Losant/losant-cli/pull/43/files#diff-38d133717a02b1c5f8f010bdbcbc0ba1f3f51d1a994feb847c829ddfd8eb4118R11 (the command description)